haskell/src/Monocle/Api/Server.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE TupleSections #-}
```

Comment:
nit: please check https://github.com/NorfairKing/haskell-dangerous-functions#--language-tuplesections-- . Perhaps this was added because of hlint, and we should configure it to ignore that rule.